presidio-anonymizer/presidio_anonymizer/entities/analyzer_result.py

```diff
@@ -108,3 +108,11 @@ def __validate_fields(self, content):
             raise InvalidParamException(
                 f"Invalid input, analyzer result start index '{start}' "
                 f"must be smaller than end index '{end}'")
+
+    def validate_position_in_text(self, text_len: int):
```

Comment:
you don't test this method

Reply:
I will add it in my next commit

Reply:
This method is currently in. a wrong place... will be moved in the next PRs.